node/node.go

```diff
@@ -64,6 +64,9 @@ type Node struct {
 	rpcAPIs       []rpc.API
 	inprocHandler *rpc.Server // In-process RPC request handler to process the API requests
 
+	nrAppName string
+	nrLicense string
```

Comment:
How about leaving comments about new variables?

Reply:
@ehnuje I removed these variables and make the node read os environment variables when NewRelic is configured